x-pack/plugins/cases/server/routes/api/configure/patch_configure.ts

```diff
@@ -10,30 +10,36 @@ import { pipe } from 'fp-ts/lib/pipeable';
 import { fold } from 'fp-ts/lib/Either';
 import { identity } from 'fp-ts/lib/function';
 
-import { CasesConfigurePatchRt, throwErrors } from '../../../../../common/api';
-import { RouteDeps } from '../../types';
-import { wrapError, escapeHatch } from '../../utils';
-import { CASE_CONFIGURE_URL } from '../../../../../common/constants';
+import {
+  CaseConfigureRequestParamsRt,
+  throwErrors,
+  CasesConfigurePatch,
+} from '../../../../common/api';
+import { RouteDeps } from '../types';
+import { wrapError, escapeHatch } from '../utils';
+import { CASE_CONFIGURE_DETAILS_URL } from '../../../../common/constants';
 
 export function initPatchCaseConfigure({ router, logger }: RouteDeps) {
   router.patch(
     {
-      path: CASE_CONFIGURE_URL,
+      path: CASE_CONFIGURE_DETAILS_URL,
       validate: {
+        params: escapeHatch,
         body: escapeHatch,
       },
     },
     async (context, request, response) => {
       try {
-        const query = pipe(
-          CasesConfigurePatchRt.decode(request.body),
+        const params = pipe(
+          CaseConfigureRequestParamsRt.decode(request.params),
```

Comment:
I added but I am not sure if it is needed. The `params` are part of the url. If a user pass additional params it will get a `404` from Kibana because the endpoint will not exists.

Reply:
Oh I missed that, you can disregard 👍